wp-content/plugins/goonj-blocks/src/render.php

```diff
@@ -36,10 +42,18 @@
 );
 
 if ( in_array( $target, array( 'collection-camp', 'dropping-center' ) ) ) :
-	$start_date = new DateTime( $action_target['Collection_Camp_Intent_Details.Start_Date'] );
-	$end_date   = new DateTime( $action_target['Collection_Camp_Intent_Details.End_Date'] );
-	$address = $action_target['Collection_Camp_Intent_Details.Location_Area_of_camp'];
-
+    if ( 'dropping-center' === $target ) {
+        $start_date = new DateTime( $action_target['Dropping_Centre.Start_Time'] );
+        $end_date   = new DateTime( $action_target['Dropping_Centre.End_Time'] );
```

Comment:
Not needed.

Reply:
@belwalshubham Why is it not needed?

@coderabbitai you also ask a counter question. Like in this case, if it is not need then it means `$action_target['Dropping_Centre.Start_Time']` and `$action_target['Dropping_Centre.End_Time']` are always going to be in the correct data format. So what things have we put in place to make sure that the date is always in correct format.

Reply:
@pokhiii 
I think we need to wrap this block under try-catch. If an invalid date string is provided, it will catch the Exception and log the error using \Civi::log()->info()